app/services/idv/phone_confirmation_session.rb

```diff
@@ -2,36 +2,47 @@
 
 module Idv
   class PhoneConfirmationSession
-    attr_reader :code, :phone, :sent_at, :delivery_method
+    attr_reader :code, :phone, :sent_at, :delivery_method, :user
 
-    def self.generate_code
-      OtpCodeGenerator.generate_alphanumeric_digits(
-        TwoFactorAuthenticatable::PROOFING_DIRECT_OTP_LENGTH,
-      )
+    def self.generate_code(user: nil)
```

Comment:
since this class is always for IDV, it seems like it should always get a `user`? Seems like less of a chance we accidentally skip a callsite
```suggestion
    def self.generate_code(user:)
```

Reply:
agreed!